pkg/dbmate/db.go

```diff
@@ -221,8 +254,67 @@ func (db *DB) openDatabaseForMigration() (Driver, *sql.DB, error) {
 	return drv, sqlDB, nil
 }
 
-// Migrate migrates database to the latest version
-func (db *DB) Migrate() error {
+// Migrate migrates database to the latest version. If withRollback is true,
+// this doesn't migrate but returns if such a migration may fail.
+func (db *DB) Migrate(withRollback bool) error {
+	if withRollback {
+		return db.migrateAndRollback()
+	}
+	return db.migrate()
+}
+
+func (db *DB) migrateAndRollback() error {
+	re := regexp.MustCompile(`^\d.*\.sql$`)
+	files, err := findMigrationFiles(db.MigrationsDir, re)
+	if err != nil {
+		return err
+	}
+
+	if len(files) == 0 {
+		return fmt.Errorf("no migration files found")
```

Comment:
can just use 
```
errors.New()
```

Reply:
This was old behavior as in https://github.com/amacneil/dbmate/blob/master/pkg/dbmate/db.go#L233

Won't changing this impact other scripts if they were greping for this?

Reply:
yeah don't worry about it this is very minor